galaxy.yml

```
@@ -2,7 +2,7 @@
 authors:
   - Ansible Security Team (https://github.com/ansible-security)
 dependencies:
-  "ansible.netcommon": ">=2.0.1"
+  "ansible.netcommon": ">=2.5.0"
```

This has to be 2.5.1 to work
```suggestion
  "ansible.netcommon": ">=2.5.1"
```